pkgpanda/__init__.py

```diff
@@ -356,7 +356,7 @@ def load(self, id: str):
         filename = os.path.join(path, "pkginfo.json")
         try:
             pkginfo = load_json(filename)
-        except FileNotFoundError as ex:
+        except OSError as ex:
```

Comment:
@branden , the `NotADirectoryError` shouldn't  have been raised here.

At line 353, we have the explicit check for  the directory presence: `if not os.path.exists(path):`